core-bundle/src/Resources/config/listener.yml

```diff
@@ -88,6 +88,11 @@ services:
         tags:
             - { name: contao.hook, hook: loadDataContainer }
 
+    Contao\CoreBundle\EventListener\DataContainer\Page\AdjustAutoforwardSubpaletteListener:
+        autoconfigure: true
```

Comment:
This should be moved to the defaults (see the `services.yml` file):

```php
services:
    _defaults:
        autoconfigure: true
```

Reply:
I wasn't sure if this would have any unintended side-effects, since the other services weren't autoconfigured before.

Reply:
Would it make sense to create another config (`listener_dca`) file for the DCA callbacks?

Reply:
I don't think so.